gspread/auth.py

```diff
@@ -274,14 +288,18 @@ def service_account(filename=DEFAULT_SERVICE_ACCOUNT_FILENAME, scopes=DEFAULT_SC
 
     :param str filename: The path to the service account json file.
     :param list scopes: The scopes used to obtain authorization.
+    :type client_factory: :class:`gspread.ClientFactory`
```

Comment:
here too, the last part of the line should be: `` :class:`gspread.client.Client` ``

Reply:
Here this is correct, and should be `ClientFactory` (which is `Type[Client]`), see:
https://docs.python.org/3/library/typing.html#typing.Type

Reply:
Let me know if you still want this changed to `Client`.